### PR TITLE
Support 'value' keyword from Kotlin 1.5

### LINKED ISSUE #1170: Support 'value' keyword from Kotlin 1.5
On the 

```
public value class TomlParser(private val config: TomlConfig) { }
```

we receive the following:

```
Warn (1:1) [WRONG_MULTIPLE_MODIFIERS_ORDER] sequence of modifier-keywords is incorrect: public should be on position 2, but is on position 1
Warn (1:8) [WRONG_MULTIPLE_MODIFIERS_ORDER] sequence of modifier-keywords is incorrect: value should be on position 1, but is on position 2
```

looks like we do not support `value` keyword

**Please also mark this inspection in the readme with Kotlin 1.6**

### PR DESCRIPTION
**Pull request by Cheshiriks**
### What's done:
* Support 'value' keyword from Kotlin 1.6
Closes #1170

### CODE REVIEW

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt [hunk 1]
```diff
@@ -129,7 +129,7 @@ enum class Warnings(
     LONG_NUMERICAL_VALUES_SEPARATED(true, "3.14.2", "long numerical values should be separated with underscore"),
     MAGIC_NUMBER(false, "3.14.3", "avoid using magic numbers, instead define constants with clear names describing what the magic number means"),
     WRONG_DECLARATIONS_ORDER(true, "3.1.4", "declarations of constants and enum members should be sorted alphabetically"),
-    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect"),
+    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect. Value identifier supported in Kotlin 1.6"),
```
Comment (petertrr) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt:132]: I don't think it should be in a warning text. We have access to version set in `diktat-analysis.yaml` and can do some logic in the rule itself

File: info/guide/guide-chapter-3.md [hunk 2]
```diff
@@ -890,7 +891,7 @@ inner
 out
 enum / annotation
 companion
-inline / noinline
+value / inline / noinline
```
Comment (petertrr) [inline info/guide/guide-chapter-3.md:894]: It's not from 1.6, we don't even support 1.6 in diktat now. Please check if it appeared in 1.4 or 1.5

**Review by petertrr — CHANGES_REQUESTED** [review]

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt (hunk 1 above)
Comment (akuleshov7) [reply diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt:132]: This is what I have asked. I need some mentions/tags of Kotlin 1.6 features

File: info/guide/guide-chapter-3.md (hunk 2 above)
Comment (akuleshov7) [reply info/guide/guide-chapter-3.md:894]: 1.5

Comment (Cheshiriks) [reply info/guide/guide-chapter-3.md:894]: Fixed all

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt (hunk 1 above)
Comment (petertrr) [reply diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt]: We can use different list of keywords based on version from config. To aggregate we could add a marker annotation like `@MinimalLanguageVersion` for rules that are only used starting with a particular version or `@OptionalLanguageVersion` for rules that have part of their functionality available starting from a particular version (like this one).

**Review by akuleshov7 — APPROVED** [review]

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt [hunk 3]
```diff
@@ -129,7 +129,7 @@ enum class Warnings(
     LONG_NUMERICAL_VALUES_SEPARATED(true, "3.14.2", "long numerical values should be separated with underscore"),
     MAGIC_NUMBER(false, "3.14.3", "avoid using magic numbers, instead define constants with clear names describing what the magic number means"),
     WRONG_DECLARATIONS_ORDER(true, "3.1.4", "declarations of constants and enum members should be sorted alphabetically"),
-    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect"),
+    WRONG_MULTIPLE_MODIFIERS_ORDER(true, "3.14.1", "sequence of modifier-keywords is incorrect. Value identifier supported in Kotlin 1.5"),
```
Comment (akuleshov7) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt:132]: available-rules.md

**Review by petertrr — APPROVED** [review]

**Pull request merged**